README.md

```diff
@@ -1,7 +1,8 @@
 # Key Links
 * [The Coding Train](https://www.youtube.com/user/shiffman/)
 * [Become a Patreon](http://patreon.com/codingtrain "Coding Train - Patreon")
-* [Suggest a Topic!](https://github.com/CodingTrain/Rainbow-Topics/)
+* [Suggest a Topic!](https://github.com/CodingTrain/Rainbow-Topics/)  
+* [Buy Merchandise](http://codingtrain.storenvy.com)
```

Comment:
Thank you! Can you change it to "coding train store" to be a little less sales-y? :)